examples/step-71/doc/results.dox

```diff
@@ -201,3 +290,220 @@ make sense:
   FEInterfaceValues class combined with MeshWorker::mesh_loop() in the
   same spirit as we used for the assembly of the linear system.
 
+
+  <h3> Results of taking <i>&gamma;</i> in the boundary assembler </h3>
+
+  In the program, we use the $2\gamma$ in the `boundary_worker` but $\gamma$
+  in the `face_worker`. Results are presented previously.
+  Here, `boundary_worker` and `face_worker` have both $\gamma$.
+  We will present numerical results.
+
+  <h3>Test results on <i>Q<sub>2</sub></i> with <i>&gamma; = p(p+1)</i> </h3>
+
+  <table align="center" class="doxtable">
+    <tr>
+     <th>Number of refinements </th><th>  $\|u-u_h^\circ\|_{L_2}$ </th><th>  Conv. rates  </th><th>  $|u-u_h|_{H^1}$ </th><th> Conv. rates </th><th> $|u-u_h|_{H^2}$ </th><th> Conv. rates </th>
+    </tr>
+    <tr>
+     <td>   2                  </td><td>   8.780e-03 </td><td>       </td><td>  7.095e-02   </td><td>           </td><td>  1.645 </td><td>   </td>
+    </tr>
+    <tr>
+     <td>   3                  </td><td>   3.515e-03   </td><td>  1.32 </td><td> 2.174e-02  </td><td>     1.70     </td><td> 8.121e-01  </td><td>  1.018  </td>
+    </tr>
+    <tr>
+     <td>   4                  </td><td>   1.103e-03   </td><td>  1.67   </td><td> 6.106e-03    </td><td>  1.83        </td><td>   4.015e-01 </td><td> 1.016  </td>
+    </tr>
+    <tr>
+     <td>   5                  </td><td>  3.084e-04  </td><td>  1.83   </td><td>  1.622e-03   </td><td>    1.91        </td><td> 1.993e-01 </td><td>  1.010   </td>
+    </tr>
+  </table>
+  We can see in this case, convergence rates satisfy theoretical expectations.
+  Comparing taking $\gamma$ and taking $2\gamma$ in the `boundary_worker`,
+  $2\gamma$ converges better.
```

Comment:
How do you determine this? I'm seeing the following errors for the original case on the finest mesh:
  3.277e-4   1.666e-3   1.981e-1
and you have here
  3.084e-4   1.622e-3   1.993e-1
So they are almost the same.